lib/src/code_generator/library.dart

```diff
@@ -93,10 +93,23 @@ class Library {
 
   /// Formats a file using `dartfmt`.
   void _dartFmt(String path) {
-    final result = Process.runSync('dartfmt', ['-w', path],
-        runInShell: Platform.isWindows);
-    if (result.stderr.toString().isNotEmpty) {
-      _logger.severe(result.stderr);
+    try {
+      final result = Process.runSync('dartfmt', ['-w', path],
+          runInShell: Platform.isWindows);
+      if (result.stderr.toString().isNotEmpty) {
+        _logger.severe(result.stderr);
+      }
+    } on ProcessException {
+      try {
+        final result = Process.runSync('flutter', ['format', path],
+            runInShell: Platform.isWindows);
+        if (result.stderr.toString().isNotEmpty) {
+          _logger.severe(result.stderr);
+        }
+      } on ProcessException {
+        _logger.severe(
+            "Couldn't format bindings, unable to call 'dartfmt' or 'flutter format'.");
+      }
     }
```

Comment:
If you want to find something in the current SDK, I would suggest using:
https://pub.dev/documentation/cli_util/latest/cli_util/getSdkPath.html

Then you can always find a dart-sdk..

But you could also just have a dependency on https://pub.dev/packages/dart_style which is the library version.